lib/cleanup/destroyable_records.rb

```diff
@@ -58,15 +58,17 @@ def print_data
   end
 
   def destroy_records
-    stdout.puts 'Destroying integration usages'
-    integration_usages.each do |integration_usage|
-      integration_usage.destroy!
-    end
-    integration.reload
+    if integration
+      stdout.puts 'Destroying integration usages'
+      integration_usages.each do |integration_usage|
+        integration_usage.destroy!
+      end
+      integration.reload
 
-    stdout.puts "Destroying integration with issuer #{integration.issuer}"
-    integration.destroy!
-    service_provider.reload
+      stdout.puts "Destroying integration with issuer #{integration.issuer}"
+      integration.destroy!
```

Comment:
Should this bit live inside the `if integration` since we have the check for it?

Reply:
it does I think? unless I am missing something...

Reply:
`end` is on line 71

Reply:
Oh yes it does! I saw the `end` from the `do` and got confused!